image-component.code-workspace

```diff
@@ -0,0 +1,8 @@
+{
+	"folders": [
+		{
+			"path": "."
+		}
+	],
+	"settings": {}
+}
```

Comment:
Same as with the tasks.json file, I don't think this should be included in the scaffold. If you think it should be then I think we need to approach it as a separate issue/PR.

Reply:
Absolutely. Removed.